### PR TITLE
set wrap to make it more user-friendly on small screen

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

**Comment by dyng** [comment]
screen shot 2016-03-30 at 17 31 09

I think a wrapped line is some bit... **ugly**. (partly because the indention is not correctly placed between line start and the line number)

I think there is a better idea that adding a hook method after window initialization in that user can customize window as they like.

**Comment by spacewander** [comment]
Just update my pr.
@dyng Do you get time to have a review?

File: autoload/ctrlsf/win.vim [hunk 1]
```diff
@@ -119,6 +119,13 @@ func! s:InitMainWindow() abort
     setl textwidth=0
     setl nospell
     setl nofoldenable
+    if exists("*g:CtrlSFCustomizeWindowFunction")
+        if exists("b:ctrlsf_customized_window_function_called")
+            return
+        endif
+        let b:ctrlsf_customized_window_function_called = 1
+        silent! call g:CtrlSFCustomizeWindowFunction()
+    end
```
Comment (dyng) [inline autoload/ctrlsf/win.vim]: I think this is better to be placed **right before** line 148 `let b:ctrlsf_initialized = 1`, let user can overwrite every aspect of default window initialization.

File: autoload/ctrlsf/win.vim [hunk 2]
```diff
@@ -119,6 +119,13 @@ func! s:InitMainWindow() abort
     setl textwidth=0
     setl nospell
     setl nofoldenable
+    if exists("*g:CtrlSFCustomizeWindowFunction")
+        if exists("b:ctrlsf_customized_window_function_called")
```
Comment (dyng) [inline autoload/ctrlsf/win.vim]: this callback should be considered as a part of initialization, I think no need to add this variable besides of `b:ctrlsf_initialized`.

File: autoload/ctrlsf/win.vim [hunk 3]
```diff
@@ -119,6 +119,13 @@ func! s:InitMainWindow() abort
     setl textwidth=0
     setl nospell
     setl nofoldenable
+    if exists("*g:CtrlSFCustomizeWindowFunction")
```
Comment (dyng) [inline autoload/ctrlsf/win.vim]: Maybe renaming this variable to `g:CtrlSFAfterMainWindowInit` should be better? We may add callback of `g:CtrlSFAfterPreviewWindowInit` or `g:CtrlSFBeforeMainWindowInit` or something else.

**Comment by dyng** [comment]
@spacewander Thank you for your pull request! I have reviewed, please take a look at those comments.

**Comment by spacewander** [comment]
@dyng I have updated my code again.

**Comment by dyng** [comment]
A tiny issue that variable name `g:CtrlSFAftermainWindowInit` is not strict camel-cased, but I'll fix it later.

Thank you for your contribution!

**Pull request merged**